containers/src/Data/IntMap/Internal.hs

```diff
@@ -328,8 +330,8 @@ import Text.Read
 import Language.Haskell.TH.Syntax (Lift)
 -- See Note [ Template Haskell Dependencies ]
 import Language.Haskell.TH ()
-#endif
 import qualified Control.Category as Category
+#endif
```

Comment:
Does MicroHs not support `Control.Category`? It's quite an old API; wasn't it supported by implementations other than GHC in the past?

Reply:
There's no Control.Category, but that's just my laziness.